packages/flutter/lib/src/widgets/scroll_activity.dart

```diff
@@ -526,8 +531,12 @@ class BallisticScrollActivity extends ScrollActivity {
   BallisticScrollActivity(
     ScrollActivityDelegate delegate,
     Simulation simulation,
-    TickerProvider vsync,
-  ) : super(delegate) {
+    TickerProvider vsync, {
+    double initVelocity = 0.0,
+    double initPosition = 0.0,
```

Comment:
Somewhere we need to document what these parameters mean. Either in the constructor doc, or - if we can make them final as suggested below - you could make them public and document on there what they mean.

Reply:
I am especially curious to learn why these have the "init" prefix...